tests/queries/0_stateless/02251_last_day_of_month.sql

```diff
@@ -44,3 +44,12 @@ SELECT toLastDayOfMonth(date_value), toLastDayOfMonth(date_time_value), toLastDa
 WITH
     toDate('2020-12-12') AS date_value
 SELECT last_day(date_value), LAST_DAY(date_value);
+
+-- boundaries
+WITH
+    toDate('1070-01-01') AS date_value,
```

Comment:
`toDate()` is clamped to `1070-01-01` to `1970-01-01` ... to simplify, we could use `1970-01-01` directly.

Reply:
Doesn't need an explicit test but would you like to check that all days in January 1900 work? (these were broken in my local tests). February 1900 was fine.

Reply:
> `toDate()` is clamped to `1070-01-01` to `1970-01-01` ... to simplify, we could use `1970-01-01` directly.

It's a typo. It should be 1970 (not 1070).